ci-operator/step-registry/hypershift/mce/install/hypershift-mce-install-commands.sh

```diff
@@ -212,3 +219,24 @@ if [ $_configReady -eq 0 ]; then
   exit 1
 fi
 echo "Configuring the hosting service cluster Succeeded!"
+
+echo "check pods"
+_podsReady=0
+set +e
+for ((i=1; i<=10; i++)); do
+  ocm_count=$(oc get pod -n open-cluster-management-agent-addon --no-headers | grep -v Running | grep -v Completed | wc -l)
+  me_count=$(oc get pod -n multicluster-engine --no-headers | grep -v Running | grep -v Completed | wc -l)
+
```

Comment:
Is there a crd and condition we can wait on instead of looking at pod status?

Reply:
Sure! @rokej  @dtthuynh, Could you help me to answer?

Reply:
@LiangquanLi930 Use this command to check that MCE is available. The output should be `True`

```
oc get mce multiclusterengine -o jsonpath='{.status.conditions[?(@.type=="Available")].status}'
```

Use this command to check that the hypershift-addon is available. Available should be `True` and Degraded should be `False`

```
oc get managedclusteraddon hypershift-addon -n local-cluster -o jsonpath='{.status.conditions[?(@.type=="Available")].status}'`
oc get managedclusteraddon hypershift-addon -n local-cluster -o jsonpath='{.status.conditions[?(@.type=="Degraded")].status}'`
```